tensorflow/c/eager/c_api_test_util.cc

```diff
@@ -88,6 +88,20 @@ TFE_TensorHandle* TestMatrixTensorHandle(TFE_Context* ctx) {
   return th;
 }
 
+TFE_TensorHandle* TestMatrixTensorHandleWithInput(TFE_Context* ctx, float data[]) {
+  int64_t dims[] = {2, 2};
```

Comment:
Sorry I missed this earlier. `dims` should be an argument as well since `data` could have variable size.

Reply:
Got it, how much error handling is necessary in a context like this? For util test functions like these is it safe to assume that the user is verifying that their dimensions and data match up?

Reply:
Yeah, I think that is a fair assumption since this is just a test utiliity.